viewer/js/gis/dijit/LayerLoader.js

```diff
@@ -792,12 +792,16 @@ function (declare, _WidgetBase, _TemplatedMixin, _WidgetsInTemplateMixin, Dialog
                 encodedSearchTerms +
                 '"^100+OR+description:"' +
                 encodedSearchTerms +
-                '"^50+OR+name:' +
+                '"^50+OR+topic:"' +
+                encodedSearchTerms +
+                '"^75+OR+name:' +
                 encodedSearchTerms +
                 '+OR+longName:' +
                 encodedSearchTerms +
                 '+OR+description:' +
                 encodedSearchTerms +
+                '+OR+topic:' +
+                encodedSearchTerms +
```

Comment:
This, and the next change in this file, are the only thing really relevant to this pull request.